.azure/lint-linux.yml

```
@@ -0,0 +1,55 @@
+parameters:
+  - name: "pythonVersion"
+    type: string
+    displayName: "Version of Python to use"
+
+jobs:
+  - job: 'Lint'
+    pool: {vmImage: 'ubuntu-latest'}
+
+    variables:
+      PIP_CACHE_DIR: $(Pipeline.Workspace)/.pip
+
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: ${{ parameters.pythonVersion }}
```

```suggestion
          versionSpec: '${{ parameters.pythonVersion }}'
```

For consistency with `docs-linux.yml`.